libs/langchain/langchain/llms/bedrock.py

```diff
@@ -74,6 +104,15 @@ class BedrockBase(BaseModel, ABC):
     endpoint_url: Optional[str] = None
     """Needed if you don't want to default to us-east-1 endpoint"""
 
+    streaming: bool = False
+    """Whether to stream the results."""
+
+    provider_stop_sequence_key_name_map: Mapping[str, str] = {
```

Comment:
i left it a class variable in case it need to be overridden in the future, or if new providers are added.

Reply:
i left it a class variable in case it need to be overridden in the future, or if new providers are added.

Reply:
lets say its a bedrock llm object, it should not have information required by other bedrock model providers.. for it, what matters is the stop sequence key for its own implementation. IMO, it should not be a class variable because let's say Anthropic Bedrock - it will have this map for all other unrelated providers when this object should just have information about itself.